### PR TITLE
feat: update DA inputs

### PR DESCRIPTION
**Pull request by akocbek**
### Description
Issue: https://github.com/terraform-ibm-modules/terraform-ibm-event-notifications/issues/172

### Release required?

- [ ] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

File: solutions/standard/catalogValidationValues.json.template [hunk 1]
```diff
@@ -3,6 +3,5 @@
   "resource_group_name": $PREFIX,
   "tags": $TAGS,
   "existing_kms_instance_crn": "crn:v1:bluemix:public:hs-crypto:us-south:a/abac0df06b644a9cabc6e44f55b3880e:e6dce284-e80f-46e1-a3c1-830f7adff7a9::",
```
Comment (ocofaigh) [inline solutions/standard/catalogValidationValues.json.template:5]: oh, this should actually be changed to:
`"existing_kms_instance_crn": $HPCS_US_SOUTH_CRN`

File: solutions/standard/main.tf [hunk 2]
```diff
@@ -5,29 +5,33 @@
...
+  en_kms_key_id                    = local.existing_kms_root_key_id != null ? local.existing_kms_root_key_id : module.kms[0].keys[format("%s.%s", var.en_key_ring_name, var.en_key_name)].key_id
 }
 
 # KMS root key for Event Notifications
 module "kms" {
   providers = {
     ibm = ibm.kms
   }
-  count                       = var.existing_kms_root_key_id != null ? 0 : 1 # no need to create any KMS resources if passing an existing key
+  count                       = local.existing_kms_root_key_id != null ? 0 : 1 # no need to create any KMS resources if passing an existing key
```
Comment (ocofaigh) [inline solutions/standard/main.tf:29]: probably safer to use `var.existing_kms_root_key_id` here

**Review by ocofaigh — CHANGES_REQUESTED** [review]
left some comments

File: solutions/standard/main.tf (hunk 2 above)
Comment (akocbek) [reply solutions/standard/main.tf:29]: actually we removed `var.existing_kms_root_key_id` https://github.com/terraform-ibm-modules/terraform-ibm-event-notifications/pull/178/files#diff-e58a8d0a4f1a1a5eb0362e469eaf5ad7622f6dcb6f74df4683b3d8713c26ada1L85

File: solutions/standard/catalogValidationValues.json.template (hunk 1 above)
Comment (akocbek) [reply solutions/standard/catalogValidationValues.json.template:5]: done

File: solutions/standard/main.tf (hunk 2 above)
Comment (ocofaigh) [reply solutions/standard/main.tf:29]: sorry I meant `existing_kms_root_key_crn`

File: solutions/standard/catalogValidationValues.json.template [hunk 3]
```diff
@@ -2,7 +2,6 @@
   "ibmcloud_api_key": $VALIDATION_APIKEY,
   "resource_group_name": $PREFIX,
   "tags": $TAGS,
-  "existing_kms_instance_crn": "crn:v1:bluemix:public:hs-crypto:us-south:a/abac0df06b644a9cabc6e44f55b3880e:e6dce284-e80f-46e1-a3c1-830f7adff7a9::",
-  "kms_region": "us-south",
+  "existing_kms_instance_crn": $HPCS_US_SOUTH_CRN
```
Comment (ocofaigh) [inline solutions/standard/catalogValidationValues.json.template:5]: your missing the comma

Comment (akocbek) [reply solutions/standard/catalogValidationValues.json.template]: thanks, updated

**Review by ocofaigh — APPROVED** [review]

**Pull request merged**